src/ansys/aedt/core/application/design.py

```diff
@@ -3198,15 +3199,17 @@ def clean_proj_folder(self, directory=None, name=None):
             ``True`` when successful, ``False`` when failed.
 
         """
-        if not name:
+        if name is None:
             name = self.project_name
-        if not directory:
+        if directory is None:
             directory = self.results_directory
         self.logger.info("Cleanup folder %s from project %s", directory, name)
-        if os.path.exists(directory):
-            shutil.rmtree(directory, True)
-            if not os.path.exists(directory):
-                os.makedirs(directory)
+
+        input_dir_path = Path(directory)
+        if input_dir_path.exists():
+            shutil.rmtree(input_dir_path, True)
+            if not os.path.exists(input_dir_path):
```

Comment:
shouldn't os.path be avoided also here?
```suggestion
            if not input_dir_path.exists():
```